Security/TwoFactor/AuthenticationContextInterface.php

```diff
@@ -63,4 +63,11 @@ public function isAuthenticated();
      * @param bool $authenticated
      */
     public function setAuthenticated($authenticated);
+
+    /**
+     * Is authentication request.
+     *
+     * @return bool
+     */
+    public function isAuthenticationTry();
```

Comment:
Method has been removed form interface. Test is done in [TwoFactorProviderRegistry](https://github.com/scheb/two-factor-bundle/pull/61/files#diff-23e70c06a58c1b5929dbbc37e1fb5433R96)